doc/classes/@GDScript.xml

```diff
@@ -866,9 +866,10 @@
 			<argument index="0" name="s" type="float">
 			</argument>
 			<description>
-				Returns sign of [code]s[/code] -1 or 1.
+				Returns sign of [code]s[/code], -1 or 1. Returns 0 if [code]s[/code] equals to 0.
```

Comment:
- change "sign" to "the sign"
- change , to :
- "equals to 0" doesn't quite sound right. Better: "if [code]s[/code] is 0."